spec/tasks/data_migraitons_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
```

Comment:
there is a typo in filename. Also same file already exists in spec folder, so I suggest it should be renamed